schemas/metaschema/DatabaseField.schema

```diff
@@ -16,35 +16,43 @@
   comment: { domain: 'Text' },
 
   Check: Validate(data => {
+    let kind = data.constructor.name;
+    if (kind === 'Function') return true;
+
     const isDomain = !!data.domain;
     const isCategory = !!data.category;
 
     // Required just one of fields: domain or category
-    if (isDomain && isCategory) return false;
-    if (!isDomain && !isCategory) return false;
+    if (isDomain === isCategory) return false;
 
     const fields = Object.keys(data);
 
     // Check allowed fields
     const allow = allowed => fields.every(f => allowed.includes(f));
 
-    let kind = data.constructor.name;
-    if (!kind && isCategory) kind = 'Category';
+    if (kind === 'Object' && isCategory) kind = 'Category';
     if (data.expression) kind = 'Expression';
 
     const allowedFields = {
-      Include: ['category', 'comment'],
+      Include: ['category', 'comment', 'required'],
       Master: ['category', 'control', 'comment'],
       Many: ['category', 'comment'],
       Category: ['category', 'lookup', 'required', 'control', 'comment'],
-      Expression: ['domain', 'lookup', 'category', 'control', 'comment']
+      Expression: ['domain', 'lookup', 'category', 'control', 'comment'],
+      Object: [
+        'domain', 'required', 'unique', 'hidden', 'readOnly', 'default',
+        'example', 'control', 'validate', 'normalize', 'comment','index',
+      ]
```

Comment:
This doesn't seem to support `Unique` and `Index` decorators, could you add tests for those?

Reply:
This should be fixed.